middleware/redoer.py

```diff
@@ -77,6 +82,12 @@ def go():
                                   + ' for this record; dropping on the floor and moving on.')
                     time.sleep(WAIT_SECONDS)
                     continue
+                except LongRunningCallTimeoutEx as lrex:
+                    # Abandon and move on.
+                    have_rcd = 0
+                    log.error(f'{SZ_TAG} {type(lrex).__module__}.{type(lrex).__qualname__} :: '
+                              + f'Long-running Senzing add_record call exceeded {SZ_CALL_TIMEOUT_SECONDS} sec.; '
+                              + f'skipping and moving on; receipt_handle was: {receipt_handle}')
```

Comment:
I'm seeing some duplicate code between this and `consumer.py`. Should more of this logic be moved to a shared class or module of some sort?